### PR TITLE
Updated Add friend request button and notification button

### PR DESCRIPTION
**Pull request by linga66**
As mentioned in issue id #440  I have made the changes

 
Screenshot (640)
Screenshot (641)
Screenshot (642)
Screenshot (639)

### CODE REVIEW

**Comment by linga66** [comment]
Is everything okay? @narayan954

**Comment by narayan954** [comment]
> Is everything okay? @narayan954 

have you tested this?? the code breaks

**Comment by linga66** [comment]
> > Is everything okay? @narayan954 
> 
> have you tested this?? the code breaks

Yes.! Everything works good at my side.. do I need to commit again?

**Comment by narayan954** [comment]
> > > Is everything okay? @narayan954
> > 
> > 
> > have you tested this?? the code breaks
> 
> Yes.! Everything works good at my side.. do I need to commit again?



> > > Is everything okay? @narayan954
> > 
> > 
> > have you tested this?? the code breaks
> 
> Yes.! Everything works good at my side.. do I need to commit again?

When sending friend request, error happens as seen in firebase(ignore the CORS error)
image

**Comment by narayan954** [comment]
Also, there're some merge conflicts, but I can manage them. You just fix the error :)

**Comment by linga66** [comment]
> When sending friend request, error happens as seen in firebase(ignore the CORS error)

Can you tell me where the error is!? 
I'm not good at firebase.

**Comment by narayan954** [comment]
@linga66 the sender and receiver's id is the same as well as it can be seen in the screenshot.
image

File: src/pages/Profile/index.jsx [hunk 1]
```diff
@@ -65,9 +112,35 @@ function Profile() {
           });
       }
     );
-    setVisibile(false);
+    setVisible(false);
+  };
+
+<<<<<<< HEAD
```
Comment (narayan954) [inline src/pages/Profile/index.jsx:118]: why is this here, isn't that causing syntax error? have you checked/tried running?

File: src/pages/Profile/index.jsx [hunk 2]
```diff
@@ -65,9 +112,35 @@ function Profile() {
...
+        });
+      })
+      .catch((error) => {
+        enqueueSnackbar(error.message, {
+          variant: "error",
+        });
+      });
   };
 
+>>>>>>> 7f4b99c1f5b1a5f2d72f950522421c807e28f26f
```
Comment (narayan954) [inline src/pages/Profile/index.jsx:143]: this must be causing error as well

**Review by narayan954 — CHANGES_REQUESTED** [review]
while resolving conflict, you didn't remove the merge conflict indicators and they're causing syntax errors. Please correct them and check for any syntax error.

File: src/App.jsx [hunk 3]
```diff
@@ -10,6 +10,7 @@ import LoginScreen from "./pages/Login";
 import Modal from "@mui/material/Modal";
 import Navbar from "./components/Navbar";
 import NotFoundPage from "./components/NotFound";
+import Notifications from "./components/Notifications";
```
[the remark this answers is not in the document]
Comment (linga66) [reply src/App.jsx:13]: I made notification same as favorites page. Not as Main page but as a Component

[the remark this answers is not in the document]
Comment (narayan954) [reply src/App.jsx:13]: right, but conventionally it should be in pages folder as it's now a page completely, right?

File: src/pages/Profile/index.jsx (hunk 2 above)
Comment (linga66) [reply src/pages/Profile/index.jsx]: Done

File: src/pages/Profile/index.jsx (hunk 1 above)
Comment (linga66) [reply src/pages/Profile/index.jsx]: Fixed

File: src/App.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (linga66) [reply src/App.jsx:13]: The Favorites page looks like a component, not a page, same as the notification page

[the remark this answers is not in the document]
Comment (narayan954) [reply src/App.jsx:13]: alright

**Comment by linga66** [comment]
Are there any other changes for me to solve?

**Comment by narayan954** [comment]
> Are there any other changes for me to solve?

I'll review this as soon as I get the time, please wait.

**Comment by linga66** [comment]
Is everything okay?

**Comment by linga66** [comment]
Is there any problem, why is my pr not being merged? @narayan954

**Comment by narayan954** [comment]
> Is there any problem, why is my pr not being merged? @narayan954

Hi @linga66 , actually I'm quite busy right now due to my internship and college training... I'll review it asap. I've added the labels so you can get the points. Did you not get the points?

**Comment by linga66** [comment]
No I didn't get any points

**Comment by narayan954** [comment]
> No I didn't get any points

oh ok, I'll merge this asap, your points are safe

**Review by narayan954 — APPROVED** [review]
looks ok for now, function is lagging some implementation and testing

**Comment by narayan954** [comment]
I've reduced the points as you didn't test anything, and imports were missing/incorrect.

**Comment by linga66** [comment]
> looks ok for now, function is lagging some implementation and testing

why is the label 3 removed

**Comment by narayan954** [comment]
> > > When sending friend request, error happens as seen in firebase(ignore the CORS error)
> > 
> > 
> > Can you tell me where the error is!?
> > I'm not good at firebase.
> 
> you didn't import firebase. follow the implementation of imgupload component. Also, even after sending the friend request, it doesn't show up in other users notification.

I explicitly mentioned you're missing this import , still you didn't fix that
